### ISSUE TITLE
Entering zip code instead of city pulls wrong information

### ISSUE THREAD

**Issue by cary-williams** [mikerah13/node-weather-app#1]
Entering a US zip code on the /city page will pull data not related to the zip code. 
Example: 
enter 39402 instead of Hattiesburg and get results for Current Weather for Poltava

**Comment by MichealGoldman** [comment]
Yes, I see that if entering zip code on the city page, it is interpreting the zip as the city code for the openweathermap.org api. Looking into this now
